samples/3_map_state_simple/src/app/app.module.ts

```diff
@@ -0,0 +1,34 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+import {RouterModule} from '@angular/router';
```

Comment:
Grain of salt: I'm a little _too_ focused on formatting.  Perhaps add spaces around all `{ TheThing }` imports throughout the files?

Reply:
Now that you mention it, I find it irksome.  I'll clea this up later today.

Reply:
Oh no, it appears I've sown a seed of doubt...my apologies.  😄